Makefile

```diff
@@ -256,7 +256,6 @@ bump_metallb: ## Bumps metallb commit ID and creates manifests. It also validate
 	hack/bump_metallb.sh
 	$(MAKE) bin
 	$(MAKE) bundle-release
-	$(MAKE) test
```

Comment:
why this was removed?

Reply:
"Also deleted the make test from the bump_metallb, because we want the workflow not to fail and to open PR even if the unit tests are failing, so we could catch and fix that."